config/routes.rb

```diff
@@ -1,3 +1,7 @@
 Rails.application.routes.draw do
+  devise_for :users, :controllers => {registrations: "registrations"}
+  resources :posts
```

Comment:
The routes are restricted to some actions as required [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/posts_routes_requirement.png)